packages/ui-components/src/components/layout/Dialog.vue

```diff
@@ -70,13 +85,23 @@ const props = defineProps<{
    * Prevent modal from closing when the user clicks outside of the modal or presses Esc
    */
   preventCloseOnClickOutside?: boolean
+  title?: string
+  buttons?: Array<{ text: string; props: Record<string, unknown>; onClick: () => void }>
 }>()
 
 const open = computed({
   get: () => props.open,
   set: (newVal) => emit('update:open', newVal)
 })
 
+const dialogPanelClasses = computed(() => {
+  return [
+    'transform rounded-lg overflow-hidden text-left shadow-xl transition-all relative',
+    widthClasses.value,
+    props.title || props.buttons ? 'bg-white' : 'bg-foundation'
```

Comment:
Why are we forcing it to be `bg-white`? I'd think that `bg-foundation` is fine already, but also `bg-white` isn't a semantic color and thus isn't good look in dark theme

Reply:
The Dialog in the designs have a #ffffff background. The header and footer of the dialog is bg-foundation, so when these are added I made the background white.

I can revert this to just bg-foundation and mention it to Agi when she is back.

Reply:
Fixed